src/plots/stackedBarPlot.ts

```diff
@@ -151,7 +151,7 @@ export class StackedBar<X, Y> extends Bar<X, Y> {
       if (maximum.extent !== baselineValue) {
         // only draw sums for values not at the baseline
 
-        const text = this.labelFormatter()(maximum.extent);
+        const text = this.labelFormatter()(maximum.extent, undefined, undefined, undefined);
```

Comment:
reconciliation is hard. we could do an array of datums, but that still means they have to check the args. could we use an additional formatter?? `extremaFormatter`, but even just providing of the datums at random seems slightly better. It would probably make sense to use whatever dataset is at the top of the stack